LiveSafe/static.livesafemobile.com
# LS-75:  S3 redirect for static.livesafemobile.com

- index.html and error.html for root of static domain
- js to template into error.html for redirection
- tasks to test and deploy assets to S3
### Testing
1. Delete index.html and error.html from S3
2. run `npm run deploy`
3. See that index.html and error.html are now on S3
